random-source/random-source.cabal

```diff
@@ -1,5 +1,5 @@
 name:                   random-source
-version:                0.3.0.8
+version:                0.3.0.9
```

Comment:
This should be at least `0.3.0.11`, see [hackage](https://hackage.haskell.org/package/random-source)

Reply:
Done